src/styles/_fonts.scss

```diff
@@ -1,33 +1,26 @@
-/* signika-300 - latin */
 @font-face {
+  font-family: "Supreme-Variable";
+  src: url("./fonts/Supreme/Supreme-Variable.woff2") format("woff2"),
+    url("./fonts/Supreme/Supreme-Variable.woff") format("woff");
+  font-weight: 100 800;
```

Comment:
For whatever reason, our (b, strong) rules define the bold text used in MDX renders as `bolder` not `bold` - meaning they end up with a weight of 400, which is *very* hard to distinguish from 300 in Supreme-Variable. Observe:

Signika (current):
![image](https://user-images.githubusercontent.com/63653723/153656786-0883a2df-fb02-4d2c-a187-588799f8c6b3.png)

Supreme: 
![image](https://user-images.githubusercontent.com/63653723/153656816-745d762a-b05b-48a2-a9c3-904092863400.png)

(observed in FF and Chrome on Windows 10)

Reply:
(ah, looks like `b, strong { font-weight: bolder; }` is a bootstrap thing)

Reply:
Oo good catch, I'll fix that, thanks!